music21/romanText/tsvConverter.py

```diff
@@ -173,9 +175,8 @@ def __init__(self):
         self.numeral = None
         self.relativeroot = None
         self.representationType = None  # Added (not in DCML)
-        self.extra = None
+        self.extra = {}
```

Comment:
type as `t.Dict[str, str]` -- which would catch a bug below.

Reply:
I typed this as you request and mypy still doesn't object to the code below, not sure why.